openedx/features/course_experience/static/course_experience/js/spec/LatestUpdate_spec.js

```diff
@@ -0,0 +1,30 @@
+/* globals $, loadFixtures */
+
+import 'jquery.cookie';
+import { LatestUpdate } from '../LatestUpdate';
+
+
+describe('LatestUpdate tests', () => {
+  function createLatestUpdate() {
+    new LatestUpdate({ messageContainer: '.update-message', dismissButton: '.dismiss-message button' }); // eslint-disable-line no-new
+  }
+  describe('Test cookies', () => {
+    beforeEach(() => {
+      $.cookie('update-message', '');
+      loadFixtures('course_experience/fixtures/latest-update-fragment.html');
+    });
+
+    it('Test dismiss button', () => {
+      createLatestUpdate();
+      expect($('.update-message').attr('style')).toBe(undefined);
+      $('.dismiss-message button').click();
+      expect($('.update-message').attr('style')).toBe('display: none;');
+    });
+
+    it('Test cookie hides update', () => {
+      $.cookie('update-message', 'hide');
```

Comment:
The thing is that the message hidden on object creation when the cookie is set. Setting the cookie after creation won't hide the message. That's why I couldn't leave the message creation in the `beforeEach`. The only other pattern that would work is:

1. Create message
2. Expect to be seen
3. Create cookie
4. Create message again
5. Expect to be hidden.

The behavior of 1-2 is already covered by the first test, so I didn't think it makes sense to duplicate it here.